### PR TITLE
tribute page

### PR DESCRIPTION
**Pull request by shafiuyushawu**
# Tribute page for funeral

- tribute page
- style for tribute
- readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,236 @@
...
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+<a href='https://github.com/beheshtaquraishi/tribute-'>beheshtaquraishi<a>
+
+<p align="right">(<a href="#readme-top">back to top</a></p>
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./MIT.md) licensed.
```
Comment (shafiuyushawu) [inline README.md:234]: Good job is done for adding licenses to your README.md file, but I will suggest if you include MIT.md file to your root folder to link `This project is [MIT](./MIT.md) licensed.`

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,9 @@
+#image {
+  max-width: 100%;
+  height: auto;
+  display: block;
+}
+#title {
```
Comment (shafiuyushawu) [inline styles.css]: Kindly leave spaces between your CSS selectors in your styles.css

File: index.html [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+                    <br>
+                    -- Indian Prime Minister Manmohan Singh
+                </p>
+            </div>
+            <h4>If you have time, you should read more about this incredible human being on his <a id="tribute-link"
+                    href="https://www.freecodecamp.org/learn/responsive-web-design/responsive-web-design-projects/build-a-tribute-page"
+                    target="_blank">Wikipedia entry</a>.</h4>
+        </div>,
+
+    </main>
```
Comment (shafiuyushawu) [inline index.html]: A good job is done for using appropriate HTML semantics. I still think you can nicely indent your HTML tags for a clear view.

**Review by shafiuyushawu — COMMENTED** [review]
Hello @shafiuyushawu 

download

### **Here are some few highlights.**

- You have linters errors in your CSS stylesheet.
- Good HTML semantics used
- Professional readme page.

But there are some few touches to be done to call it a success.
Kindly consider below comments to make your project lit.

Thanks you!

File: styles.css (hunk 2 above)
Comment (Shedrack-Sunday) [reply styles.css]: - [ ] You did well pointing out slack for best practices, however, I suggest you give reasons why leaving such space is important. that creates a lasting impact on the potential project owner for future occurrences. 👍 
- [ ] Also, consider using emojis 😃 , as they add an expressive friendly intent. Good job 👏

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hi @shafiuyushawu  👋    ,


CHANGES REQUIRED ♻️

Great work on making considerable progress in implementing the changes from the project requirements so far 👏
However, there are still a few which aren't been addressed yet. But its little and you should get approved after that 💯 




Highlight
✅ Proper setup
✅ Descriptive commit messages


Required Changes ♻️

- [x] A few points of note that I made regarding the review you made has been highlighted in the comments. suggestion evolves around giving a reason for a "why" and adding spice t the tune of the message with some beautiful emojis such as 👍 😃 😉 💯 etc
 
Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg"
+                alt="image">
+            <figcaption id="img-caption"> Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to
+                increase wheat yields - part of his life-long war on hunger. </figcaption>
+            </img>
+            <div id="tribute-info">
+                <h3> Here's a time line of Dr. Borlaug's life:</h3>
+                <ul>
+
+
```
Comment (shafiuyushawu) [inline index.html:37]: I suggest you remove the empty blank spaces in your code as it may expose your code 😊

File: index.html [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+        max_width: 100%;
+
+
+    }
+
+    #image {
+        max-width: 100%;
+        width: 100%;
+    }
+</style>
```
Comment (shafiuyushawu) [inline index.html:21]: Kindly move your inline style to your `style.css` to make your code looks simplified and professional👌

**Review by shafiuyushawu — COMMENTED** [review]
@shafiuyushawu 

157397398-00904e26-60ec-46cd-9510-240661c250ef

### **HIGHLIGHTS**

✅ Your code passed linters 👍
✅ Good GitHub flow 👍

FEW CHANGES REQUIRED ♻️

There are some few changes needed to be corrected even thou they are optional. Optional requirement does not mean your code is bad, it is just an extra correction to make your code looks professional.

Kindly consider below comments.

**Review by Shedrack-Sunday — APPROVED** [review]
Hi @shafiuyushawu   👋 
 
 # STATUS: APPROVED ✔️  🍾  🥳

 Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: Congratulations! 🎉

 
 
 ## Highlights
- Proper use of Github features 💯 
- Nice review ✔️ 
 
 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._
 

 Cheers and Happy coding!👏👏👏
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**